swagger.yaml

```diff
@@ -3893,6 +3922,18 @@ definitions:
         type: string
         description: Human-readable description of the error.
 
+  cluster_default_config:
+    type: object
+    properties:
+      cluster_network_cidr:
+        type: string
+      cluster_network_host_prefix:
+        type: integer
+      service_network_cidr:
+        type: string
+      ntp_source:
+        type: string
```

Comment:
Please add "pattern" for strings where you can - you can find the values in this file where the properties are defined.  Also cluster_network_host_prefix has a minimum and maximum.

Reply:
Is it needed in a reply definition? 
Usually it is used for user input no?

Reply:
I was on the fence about it but saw that we already have it elsewhere in the Swagger, and maybe some clients will use that information.